test_e2e/expected-structured-output/github/LICENSE.txt.json

```diff
@@ -1,57 +1,117 @@
 [
   {
+    "type": "Title",
     "element_id": "52585ab256e2832166ca185be6c76cc9",
+    "text": "Downloadify: Client Side File Creation JavaScript + Flash Library",
     "metadata": {
-      "filetype": "text/plain",
       "languages": [
         "eng"
-      ]
-    },
-    "text": "Downloadify: Client Side File Creation JavaScript + Flash Library",
-    "type": "Title"
+      ],
+      "filetype": "text/plain",
+      "data_source": {
+        "url": "https://api.github.com/repos/dcneiner/Downloadify/git/blobs/2c4f1ab8689a6dfef4ee7d13d4d935cb6663a7e4",
+        "version": "W/\"a75db90ed53f327faf43999f4719f8d200fb19cf1db08fe6ace57e88426ec4f4\"",
+        "record_locator": {
+          "repo_path": "dcneiner/Downloadify",
+          "file_path": "LICENSE.txt"
+        },
+        "date_created": null,
```

Comment:
Seems like you have to rerun your local test with OVERWRITE_FIXTURES=true since you added date_created and date_modified to --metadata-exclude